modules/storages/app/common/storages/snitch.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+#-- copyright
+#++
+
+module Storages
+  module Snitch
```

Comment:
I suck at naming (and I'm huge fan of analogies), hence the name. 

I considered `TaggedLogger` but that's not exactly what it does so it kinda bothered me. It doesn't provide one, just facilitates access to it. It is a thin wrapper over the Rails logger in case we want to mess up with the tags.

I know, semantics... but, IMO, code is basically semantics.

Reply:
I understand your reasoning... maybe, since it is a module, it could also be something like "TaggedLogging". Or maybe someone else has a better idea :slightly_smiling_face:

Reply:
renamed! :D